pangeo_forge_orchestrator/model_builders.py

```diff
@@ -263,16 +265,32 @@ def _update(
             session: Session = Depends(self.get_session),
             id: int,
             model: self.models.update,  # type: ignore
+            authorized_user=Depends(self.auth_dependency),
```

Comment:
```suggestion
            authorized_user: APIKeyRead = Depends(self.auth_dependency),
```
Or can this sometimes be something other than `APIKeyRead`? Guessing this type based on 

```python
    register_endpoints(
         ...
         auth_dependency=check_authentication_header_admin,
     )
```
in `api.py`.

Reply:
I am going to pass on this suggestion because importing APIKeyRead in this module would create a circular dependency. This could eventually be fixed by refactoring things a bit.